flexmeasures/data/scripts/add_scripts_path.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+current_dir=$(pwd)
+
+# Declares a function named `save_path` that takes two arguments: `path` and `extension`.
+# The function is responsible for checking if the given path is already present in the `bashrc` file.
+function save_path() {
+    path="$1"
+    extension="$2"
+
+    # Check if the path is already present in bashrc
+    if grep -qF ". $1" ~/.$2rc; then
+        echo "Path already exists in .$2rc"
+    else
+        # Add the path to bashrc
+        echo ". $1" >> ~/.$2rc
+        echo "Path added to .$2rc"
+    fi
+}
+
+# The `main` function is responsible for executing the main logic of the script.
+# It sets the `script_path` variable to the path of the "ci" directory under the current directory.
```

Comment:
I have moved the scripts from `ci` directory to `data/scripts`

Reply:
Actually they should live somewhere in the CLI package, I think.